server/src/main/java/org/elasticsearch/action/admin/cluster/desirednodes/TransportUpdateDesiredNodesAction.java

```diff
@@ -183,16 +183,22 @@ public ClusterState execute(ClusterState currentState, List<TaskContext<UpdateDe
             for (final var taskContext : taskContexts) {
 
                 final var previousDesiredNodes = desiredNodes;
+                DesiredNodes newDesiredNodes;
                 try {
-                    desiredNodes = updateDesiredNodes(desiredNodes, taskContext.getTask().request());
+                    newDesiredNodes = updateDesiredNodes(desiredNodes, taskContext.getTask().request());
```

Comment:
nit: maybe we can extract `taskContext.getTask().request()` to a variable so the code is a bit shorter?

Reply:
Done!